### PR TITLE
Add possibility of having non-technical descriptive header in addition to technical header

### PR DESCRIPTION
**Pull request by kevinkhao**

### CODE REVIEW

**Comment by kevinkhao** [comment]
Also add support for adding parents and their children in one import

File: pattern_import_export_xlsx/tests/test_pattern_import.py [hunk 1]
```diff
@@ -195,3 +207,11 @@ def disable_test_import_users_fail(self):
         self.assertEqual(ws["A1"].value, "#Error")
         self.assertTrue(ws["A2"].value)
         self.assertTrue(ws["A3"].value)
+
+    def test_import_partners_with_parents(self):
+        self._load_file("example.partners.parent.xlsx", self.ir_export_partner)
+        self.env.clear()
```
Comment (sebastienbeau) [inline pattern_import_export_xlsx/tests/test_pattern_import.py]: why to you use self.env.clear ?

File: pattern_import_export/models/ir_exports.py [hunk 2]
```diff
@@ -227,6 +238,7 @@ def _generate_import_with_pattern_job(self, patterned_import):
         res = (
             self.with_context(load_format="flatty")
             .env[self.model_id.model]
+            .with_context(pattern_import_export_model=self.model_id.model)
```
Comment (sebastienbeau) [inline pattern_import_export/models/ir_exports.py]: There is already a context before, merge it

Comment (kevinkhao) [reply pattern_import_export/models/ir_exports.py]: Done

File: pattern_import_export_xlsx/tests/test_pattern_import.py (hunk 1 above)
Comment (kevinkhao) [reply pattern_import_export_xlsx/tests/test_pattern_import.py]: Done. I presumed it was needed after seeing it once. Removed useless instances of it.

**Comment by sebastienbeau** [comment]
I have merged the PR without you last commit, please do a PR with this last commit (and the additionnal improvement)

**Pull request closed without merging**